internal/server/data/migrationdata/202207041724-postgres.sql

```diff
@@ -0,0 +1,225 @@
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 14.4 (Debian 14.4-1.pgdg110+1)
+-- Dumped by pg_dump version 14.4 (Debian 14.4-1.pgdg110+1)
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+SET default_tablespace = '';
+
+SET default_table_access_method = heap;
+
+--
+-- Name: destinations; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE testing.destinations (
+    id bigint NOT NULL,
+    created_at timestamp with time zone,
+    updated_at timestamp with time zone,
+    deleted_at timestamp with time zone,
+    name text,
+    unique_id text,
+    connection_url text,
+    connection_ca text,
+    resources text,
+    roles text
+);
+
+
+ALTER TABLE testing.destinations OWNER TO postgres;
+
+--
+-- Name: destinations_id_seq; Type: SEQUENCE; Schema: public; Owner: postgres
+--
+
+CREATE SEQUENCE testing.destinations_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE testing.destinations_id_seq OWNER TO postgres;
+
+--
+-- Name: destinations_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: postgres
+--
+
+ALTER SEQUENCE testing.destinations_id_seq OWNED BY testing.destinations.id;
+
+
+--
+-- Name: migrations; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE testing.migrations (
+    id character varying(255) NOT NULL
+);
+
+
+ALTER TABLE testing.migrations OWNER TO postgres;
+
+--
+-- Name: providers; Type: TABLE; Schema: testing; Owner: postgres
+--
+
+CREATE TABLE testing.providers (

```

Comment:
The db migration checks if the provider table exists and creates all schemas if it doesn't. This is problematic for tests since it'll create the destinations table at the current version. To work around this, the SQL needs to include the providers table. Alternatively, the migrator should check the migrations table instead.

Reply:
The postgres dump for all tables is large and most of it is irrelevant so I cherry picked certain tables. Previously, it was sufficient to just dump the providers and migrations table. This time, the dump needs to contain the destinations, providers, and migrations table

Reply:
nvm the partial dump is causing problems else where. will update the postgres sql with a full dump